docs/source/platform/schema-validation.md

```diff
@@ -147,7 +146,31 @@ After analyzing the changes against current usage metrics, Apollo will identify
 2. **Warning**: There are potential problems that may come from this change, but no clients are immediately impacted.
 3. **Notice**: This change is safe and will not break current clients.
 
-The more [performance metrics](./performance.html) that Apollo has, the better the report of these changes will become.
+<h3 id="cli-advanced">Advanced CLI Usage</h3>
+
+By default, the CLI will look at operations run within the last day in order to decide which changes are breaking. This period of time may not be large enough, so the `validationPeriod` flag accommodates custom timeframes:
```

Comment:
huh? Isn't the default to use the maximum time window by the paid-for retention of the plan? If not, then why did we change that? When you don't pass any historical parameters, that's what it does.

Reply:
We should say "the `validationPeriod` is always the duration to include prior to **now**" or the like. Let's make it clear that the "to" is always "right now" and you can't override that atm.

Reply:
The default is a day. We haven't had a chance to implement the default as the max time window yet

Reply:
Addressed both here: d86144beab8cb26492104b3204dac64248509315

I opted not to specify about defaults since it varies from plan to plan, but if you see a good way to present that information I'm open to it.

Based on previous behavior and my incorrect assumption, I'll need to update behavior of the CLI with the respective change. This is a really good catch, thank you 😄